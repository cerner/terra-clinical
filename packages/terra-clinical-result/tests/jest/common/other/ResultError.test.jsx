import React from 'react';
/* eslint-disable-next-line import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import ResultError from '../../../../src/common/other/_ResultError';

describe('ResultError', () => {
  it('should render a default ResultError', () => {
    const result = shallowWithIntl(<ResultError />).dive();
    expect(result).toMatchSnapshot();
  });
});
