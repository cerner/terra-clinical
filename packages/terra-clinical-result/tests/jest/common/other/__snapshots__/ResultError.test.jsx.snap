// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultError should render a default ResultError 1`] = `
<span
  className="system-error"
>
  <IconError
    aria-hidden="true"
    className="icon-error"
    viewBox="0 0 48 48"
    xmlns="http://www.w3.org/2000/svg"
  />
  Terra.clinicalResult.resultError
</span>
`;
